tests/builders/test_rnn.py

```diff
@@ -49,44 +51,65 @@ def rnn_match_cfg(rnn: RNN, rnn_cfg: rnn_pb2.RNN, input_features: int) -> None:
         bias += getattr(rnn.rnn, f"bias_hh_l{l}")[
             hidden_size : 2 * hidden_size
         ]
-        assert torch.allclose(bias, torch.tensor(forget_gate_bias))
+        assert torch.allclose(
+            bias, torch.tensor(forget_gate_bias).to(bias.device)
+        )
 
 
 @st.composite
 def rnn_cfg_tensors(
     draw,
-) -> st.SearchStrategy[Tuple[torch.nn.Module, rnn_pb2.RNN, torch.Tensor]]:
-    """Returns a search strategy for RNNs built from a config + valid input."""
+) -> st.SearchStrategy[
+    Tuple[torch.nn.Module, rnn_pb2.RNN, torch.Tensor, bool]
+]:
+    """Returns a search strategy of [RNNs, config, valid input, batch_first].
+    """
```

Comment:
Nit: minor rewording to squeeze this onto one line?

```
    """Returns search strategy of [RNNs, config, valid input, batch_first]."""
```

Reply:
done